app/mailers/application_mailer.rb

```diff
@@ -15,9 +15,9 @@ class ApplicationMailer < ActionMailer::Base
   )
 
   def email_from(topic = "")
-    community_name = topic.present? ? "#{SiteConfig.community_name} #{topic}" : SiteConfig.community_name
+    community_name = topic.present? ? "#{Settings::General.community_name} #{topic}" : Settings::General.community_name
```

Comment:
Should this not also be 
```suggestion
    community_name = topic.present? ? "#{Settings::Community.community_name} #{topic}" : Settings::Community.community_name
```